tests/local-suite/packaging.tests.js

```diff
@@ -0,0 +1,122 @@
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const fse = require('fs-extra');
+const testUtils = require('../utils/index');
+
+const serverlessExec = path.join(__dirname, '..', '..', 'bin', 'serverless');
+
+describe('simple packaging', () => {
```

Comment:
Can we warp this into a global describe block like `Integration test - Packaging`?